Cargo.toml

```diff
@@ -14,20 +14,20 @@ description = "Backtracking register allocator inspired from IonMonkey"
 repository = "https://github.com/bytecodealliance/regalloc2"
 
 [dependencies]
-log = { version = "0.4.8", default-features = false }
-smallvec = { version = "1.6.1", features = ["union"] }
-rustc-hash = { version = "1.1.0", default-features = false }
-slice-group-by = { version = "0.3.0", default-features = false }
+log = { version = "0.4.22", default-features = false }
+smallvec = { version = "1.13.2", features = ["union"] }
+rustc-hash = { version = "2.0.0", default-features = false }
```

Comment:
This the only change that matters. Updating minor versions in Cargo.toml.is only necessary when you are depending on new features. Cargo will pick the latest version either way (in the absence of a lockfile pinning it to an earlier version)

Reply:
Sorry yeah, just did a full `cargo upgrade` without giving it more thought 😅 updated the PR to only bump the `rustc-hash` crate